libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -1,3 +1,4 @@
+import { debounceTime } from 'rxjs/operators';
```

Comment:
I see you are importing this here, but not using it anywhere. Why did you import it? Were you planning to use it somewhere? Where could it be used?